runner.ts

```
@@ -19,9 +20,6 @@ const pjson = require("./package.json") as any;
 // (that cannot be changed/accessed by node providers)?
 const MIN_AR_BALANCE = 0.1;
 
-//TODO: make it configurable (main and token lvl)
-const DEFAULT_FETCHER_TIMEOUT = 50000; // ms
```

Removing this from node implementation (this could be used as "final fallback"). I believe that whole node configuration should result from manifest file (and not manifest file and some-magical-values-in-code) - for readability and maintainability reasons.
Any error in manifest config results in immediate process stop ("fail fast" principle)